src/applications/vaos/api/index.js

```diff
@@ -62,3 +62,15 @@ export function getFacilitiesBySystemAndTypeOfCare(typeOfCareId, facilityId) {
     }, TEST_TIMEOUT || 1000);
   });
 }
+
+export function getCommunityCare() {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve({
+        isEligible: true,
```

Comment:
I updated the api contract to remove this example data, for what it's worth. It doesn't reflect what the data we'll get back from the api is. But since I don't know exactly what that data is either, I don't think it's worth changing in your PR here.

Reply:
Ok.